stock_prices/stock_prices.py

```diff
@@ -1,9 +1,24 @@
 #!/usr/bin/python
 
 import argparse
+import math
 
 def find_max_profit(prices):
-  pass
+  profit = -math.inf
+  # Have to buy before the last number
+  # Have to sell before the last number
+  # Need to find the maximum between the minimum number and the max number
+
+  # Loop through each value in the array => i
+  #   Loop through the rest of the values in the array => j
+  #   subtract j-i to find the biggest
+  for i in range(0, len(prices)):
+    for j in range(i+1, len(prices)):
+      if prices[j]-prices[i] > profit:
+        profit = prices[j]-prices[i]
+
+  print("prices:", prices, "profit:", profit)
+  return profit
```

Comment:
oh dang this is super elegant. I legitimately can not believe this passes all the tests. lol. Only knock is those 2 nested for loops... 😂